src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -68,6 +68,15 @@ def insar_tops_burst(
     )
     results = asf_search.search(opts=opts)
 
+    ref_not_found = reference_scene not in results.__str__()
+    sec_not_found = secondary_scene not in results.__str__()
+    if ref_not_found and sec_not_found:
+        raise RuntimeError(f'ASF Search failed to find both {reference_scene} and {secondary_scene}.')
+    elif ref_not_found:
+        raise RuntimeError(f'ASF Search failed to find {reference_scene}.')
+    elif sec_not_found:
+        raise RuntimeError(f'ASF Search failed to find {secondary_scene}.')
```

Comment:
We should use a more specific exception type, but not sure what that might be off the top of my head.